src/components/POLine/POLineForm.js

```diff
@@ -243,6 +244,17 @@ class POLineForm extends Component {
                     accordionStatus={this.state.sections}
                     onToggle={this.onToggleSection}
                   >
+                    <Accordion
+                      label={<FormattedMessage id="ui-orders.line.accordion.itemDetails" />}
+                      id={ACCORDION_ID.itemDetails}
+                    >
+                      <ItemForm
+                        stripes={stripes}
+                        dispatch={dispatch}
+                        change={change}
+                        parentResources={parentResources}
```

Comment:
I think you can remove this since we don't use material types in this component. Also, you can remove `parentResources: PropTypes.shape({
      materialTypes: PropTypes.object.isRequired,
    }).isRequired,` from `ItemForm.js`